sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -604,11 +604,11 @@ export class ServiceBusAdministrationClient extends ServiceClient {
   /**
    * Updates the queue based on the queue properties provided.
    * All queue properties must be set even though only a subset of them are actually updatable.
-   * Therefore, the suggested flow is to use `getQueue()` to get the complete set of queue properties,
-   * update as needed and then pass it to `updateQueue()`.
+   * Therefore, the suggested flow is to use `getQueue()` to get the complete set of queue properties(accompanied by the _response),
```

Comment:
Is this clarification needed? Do we expect people would try to send the output without the response and therefore want to guard against that case?

Reply:
The clarification meant to clear the confusion that the `getQueue()` returns an object which not just includes properties but also the raw response.

Reply:
Then , I would rephrase to avoid calling out inner details like `_response`

> Therefore, the suggested flow is to use the output from `getQueue()`, update the desired properties in it, and then pass the modified object to `updateQueue()`.

Reply:
Done